content/en/community/hackathons/sessions/bincompat/content/Work-Items_02.-Build-app-elfloader.md

```diff
@@ -0,0 +1,40 @@
+Using `do.sh run` we ran the prebuilt images from [the `run-app-elfloader` repository](https://github.com/unikraft/run-app-elfloader).
+Let's now also build `app-elfloader`.
+
+Run the following commands:
+
+```console
+$ ./do.sh clean
+
+$ ./do.sh setup
+
+$ ./do.sh build
+
+$ ./do.sh run_built ...
+```
+
+In the last command, replace `...` with the name of the application you need to run.
+
+If you want to remove all the pesky debugging information, use `setup_plain`, such as the commands below:
+
+```console
+$ ./do.sh clean
+
+$ ./do.sh setup_plain
+
+$ ./do.sh build
+
+$ ./do.sh run_built ...
```

Comment:
Is `run_built` a command in the [`app-elfloader` script](https://github.com/unikraft-upb/scripts/blob/main/make-based/app-elfloader/do.sh#L205-L251)? It does not seem so.

Reply:
See this: https://github.com/unikraft-upb/scripts/pull/32